Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -70,6 +57,11 @@ class MessagingInAppImplementationTest: UnitTest {
         let given = String.random
 
         messagingInApp.screenViewed(name: given)
+        let expectation = XCTestExpectation(description: "Wait for 1 second")
```

Comment:
can you please explain why are we waiting for 1 second?

Reply:
We need that pause to ensure [the code completes](https://github.com/customerio/customerio-ios/blob/b850ac74bdf2b48bff78f709996d97c207b3279b/Sources/MessagingInApp/MessagingInAppImplementation.swift#L42-L44) before testing, otherwise those tests might fail due to the main thread's route setting timing. I hope it helps.